### ISSUE TITLE
Natural Diamonds image Classification on the basis of shape

### ISSUE THREAD

**Issue by kyra-09** [Niketkumardheeryan/ML-CaPsule#760]
Description :- Classification of different types of natural diamonds on the basis of shapes like round, cushion, radiant, emerald, heart, oval, etc using dl modles and comparing their preformance using different matrices such as accuracy score , confusion matrix , plotting graphs and doing EDA analysis .

dataset I'll use :- https://www.kaggle.com/datasets/harshitlakhani/natural-diamonds-prices-images/data

tools i'll use :- Keras , matplotlib , scikit-learn , tqdm , numpy etc.

Models I'll use :- Xception , VGG16 , ResNet50 etc.

I'll also use data augmentation techniques to increase the efficiency and performance of the models. (such as shearing , rotation etc.)

**Comment by kyra-09** [comment]
@Niketkumardheeryan @invigorzz313 Kindly assign this issue to me so that I can start working on it.

**Comment by kyra-09** [comment]
@Niketkumardheeryan sir can you assign this issue to me please?
